src/gulpconfig.json

```diff
@@ -81,7 +81,7 @@
         "build": ["clean", "lint", "tsd", "jsx", "tsc", "copy:html", "copy:js", "coffee", "livescript", "sass"],
         "bundle": ["clean:dist", "build"],
         "default": ["clean", "build", "test", "analyze", "jsdoc", "dist"],
-        "dist": ["clean", "build", "bundle", "libraryDist", "minify"],
+        "dist": ["clean", "build", "bundle", "libraryDist", "minify:css"],
```

Comment:
Figured for the most part apps (not libraries) would have actual css files and they would likely want minified versions of them by default.

I'm fine with removing that also, though.